### ISSUE TITLE
No longer works with Ferrari 458 Spider

### ISSUE THREAD

**Issue by Foxache** [camren-m/XboxWheelCompatibility#105]
The Ferrari 458 spider no longer appears to (atleast steam) games as a xbox controller .

Found:

 American Truck simulator doesnt recognise it as a xbox controller
Euro Truck sim aswell

**Comment by Foxache** [comment]
The given software to check sais it is connected. This may be more of a steam compatibility issue

**Comment by Zariaa27** [comment]
Did you change the settings in the game itself?

**Comment by snoozysoft** [comment]
Yeah, ETS2 doesn't recognize it as a controller and Windows seems to think a very slight turn of the wheel registers as a right click on the mouse. And since ETS2 has now shifted all controller bindings to Steam Input instead of ingame, it's now extra difficult/next to impossible to configure. Also doesn't seem to activate its auto-centering system at all which also kind of makes it more difficult to use.

**Comment by Zariaa27** [comment]
Check if the WheelCompatibility service is running. If the wheel isn't
detected at all, restart your computer with the wheel plugged in. If
this doesn't work, I don't think I can help.

If you are running the latest version, try the one before (1.2.0).

**Comment by DomTries** [comment]
Getting the same issue, service running, using V1.2.0 and tried V1.2.1 to no avail, had no problems previously until I reset my PC (which I now regret) and no longer can use the 458 Spider wheel for F1 23.

**Comment by Foxache** [comment]
It seems like with new Windows (DomTries) / Steam (Steam input doesnt fix) updates have broken this entirely, we can hope for updates and fixes although possibly unlikely.

**Comment by Foxache** [comment]
Possible fix: Uninstall drivers mentioned

**Comment by Foxache** [comment]
Possible issue: With the drop of support of Xbox Console Companion , drivers may have changed?
